examples/react-navigation/android-fragment/android/app/src/main/java/com/example/bottomnavigationviews/MainActivity.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.bottomnavigationviews
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import androidx.viewpager2.widget.ViewPager2
+import com.example.bottomnavigationviews.databinding.ActivityMainBinding
+import com.example.bottomnavigationviews.ui.home.HomeFragment
+import com.example.bottomnavigationviews.ui.react_native.ReactNativeFragment
+import com.facebook.react.ReactFragment
+import com.facebook.react.modules.core.DefaultHardwareBackBtnHandler
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class MainActivity : AppCompatActivity(), DefaultHardwareBackBtnHandler {
+
+    private lateinit var binding: ActivityMainBinding
+
+    private fun getLaunchOptions(message: String) = bundleOf("message" to message)
```

Comment:
Unless I'm missing something, the app doesn't actually appear to use these launch options?

Reply:
you tell me! :D Safe to delete line 20 then?